src/main/java/novamachina/exnihilosequentia/common/block/CrucibleBaseBlock.java

```diff
@@ -33,7 +33,11 @@ public void addProbeInfo(ProbeMode probeMode, IProbeInfo probeInfo, PlayerEntity
             if (crucibleTile.getFluidAmount() > 0) {
                 probeInfo.text(new TranslationTextComponent("waila.crucible.fluid", new TranslationTextComponent(crucibleTile.getFluid().defaultFluidState().createLegacyBlock().getBlock().getDescriptionId()), crucibleTile.getFluidAmount()));
             }
-            probeInfo.text(new TranslationTextComponent("waila.crucible.heat", crucibleTile.getHeat()));
+            Object heat = crucibleTile.getHeat();
```

Comment:
I'm not a big fan of using the Object class like this. We should actually use the type that we're returning. A tertiary operator could be used to determine what to print.